CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change log
+
+## Version 2.1.1
+
+- Blacklist/Whitelist check fix [#538](https://github.com/auth0/angular2-jwt/pull/538)
+- Refactor deep rxjs imports and use named define [#608](https://github.com/auth0/angular2-jwt/pull/608)
+- fix(rxjs): remove imports from rxjs/internal [#542](https://github.com/auth0/angular2-jwt/pull/542)
+- Added CODEOWNERS, PULL_REQUEST_TEMPLATE and ISSUE_TEMPLATE files [#590](https://github.com/auth0/angular2-jwt/pull/590), [#599](https://github.com/auth0/angular2-jwt/pull/599)
```

Comment:
This should probably be left out of a CHANGELOG since it's irrelevant to folks that are up/downgrading

Reply:
@joshcanhelp I was going back and forth on whether to include it or not. I've removed it now.